README.md

```diff
@@ -33,6 +33,24 @@ Pass in a file to run:
 npx tsx ./file.ts
 ```
 
+### Run via loader/require hook
+You can also pass `tsx` into the [`--loader`](https://nodejs.org/api/cli.html#--experimental-loadermodule) flag if using esm.
+```sh
+node --loader tsx ./file.ts
+```
+
+Or pass `tsx` into the [`--require`](https://nodejs.org/api/cli.html#-r---require-module) flag if using cjs.
+```sh
+node -r tsx ./file.ts
+```
+
+This works well for tools that allow you to pass [Node.js CLI flags](https://nodejs.org/api/cli.html) but expose their own binary to run.
+For example for usage with [mocha](https://mochajs.org) you could run the following:
+```sh
+mocha -r tsx
+```
```

Comment:
@privatenumber it does work if you use mocha in esm mode, and with our changes here it will work for cjs mode also. The only real problem is the warning that loaders are experimental which js kinda lame.